### PR TITLE
Adding a RDF file for the solr operator

### PR DESCRIPTION
**Pull request by HoustonPutman**
https://issues.apache.org/jira/browse/SOLR-15211

### CODE REVIEW

File: content/doap/solr-operator.rdf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <license rdf:resource="https://spdx.org/licenses/Apache-2.0" />
+    <name>Apache Solr Operator</name>
+    <homepage rdf:resource="https://solr.apache.org/operator" />
+    <asfext:pmc rdf:resource="https://solr.apache.org" />
+    <shortdesc>Manages Solr resources in Kubernetes</shortdesc>
+    <bug-database rdf:resource="https://github.com/apache/solr-operator/issues" />
+    <mailing-list rdf:resource="https://solr.apache.org/community.html#mailing-lists-chat" />
+    <download-page rdf:resource="https://solr.apache.org/operator/downloads.html" />
+    <programming-language>Go</programming-language>
+    <category rdf:resource="https://projects.apache.org/category/cloud" />
```
Comment (janhoy) [inline content/doap/solr-operator.rdf:35]: You can add several categories. I'd put at least `<category rdf:resource="https://projects.apache.org/category/search"/>` and `<category rdf:resource="https://projects.apache.org/category/go"/>`. You can see available categories in https://projects.apache.org/projects.html?category

File: content/doap/solr-operator.rdf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+  <Project rdf:about="https://solr.apache.org/operator">
+    <created>2021-01-14</created>
+    <license rdf:resource="https://spdx.org/licenses/Apache-2.0" />
+    <name>Apache Solr Operator</name>
+    <homepage rdf:resource="https://solr.apache.org/operator" />
```
Comment (janhoy) [inline content/doap/solr-operator.rdf:28]: You plan to make a new section in homepage? You could of course link to GitHub page as a start, and change it once the operator gets a sub section in solr homepage?

**Review by janhoy — COMMENTED** [review]
Add a few categories and reconsider what homepage to publish from day 1. Else ok.
This file can be changed any time, and changes will be picked up by ComDev's scripts every night I think.

File: content/doap/solr-operator.rdf (hunk 2 above)
Comment (HoustonPutman) [reply content/doap/solr-operator.rdf:28]: Sounds good. I am working on a sub-site, but only have a few pages ready so far. Will post that fun PR when more is complete 😅

Comment (janhoy) [reply content/doap/solr-operator.rdf:28]: Cool

File: content/doap/solr-operator.rdf (hunk 1 above)
Comment (HoustonPutman) [reply content/doap/solr-operator.rdf:35]: After looking through the tags, it looks like we should add the `database` tag for Solr.

**Review by janhoy — APPROVED** [review]

**Pull request merged**